editor/src/components/canvas/ui-jsx-canvas-renderer/scene-root.tsx

```diff
@@ -135,14 +137,19 @@ export const SceneRootRenderer = betterReactMemo(
 
     useRunSpy(scenePath, templatePath, topLevelElementName, sceneProps)
 
+    const propsWithScenePath = {
+      ...sceneProps.props,
+      [UTOPIA_SCENE_PATH]: scenePath,
+    }
```

Comment:
Previously we were providing the `scenePath` to the `ComponentRendererComponent` via the context when creating those `ComponentRendererComponent`s for each top level element. The reason being that the concept of nested "scene"s was never really a thing for us. This meant that when calling `renderCoreElement` later on down the line, the template path being passed in (`ownTemplatePath`) would only be correct for top level scenes. To support drilling into components we needed to address this, so now we pass in the actual `scenePath` via a prop here.